var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -32,14 +32,16 @@ class Libfabric(AutotoolsPackage):
     homepage = "https://libfabric.org/"
     url      = "https://github.com/ofiwg/libfabric/releases/download/v1.5.0/libfabric-1.5.0.tar.gz"
 
+    version('1.5.3', '1fe07e972fe487c6a3e44c0fb68b49a2')
     version('1.5.0', 'fda3e9b31ebe184f5157288d059672d6')
+    version('1.4.2', '2009c8e0817060fb99606ddbf6c5ccf8')
 
     fabrics = ('psm',
                'psm2',
                'sockets',
                'verbs',
                'usnic',
-               'mxm',
+               'mlx',
                'gni',
                'xpmem',
```

Comment:
I would check this with project maintainers but in the git I do not see xpmem code. Maybe there are some proprietary tarballs

Reply:
The only xpmem code I can find is
https://code.google.com/archive/p/xpmem/downloads
https://github.com/hjelmn/xpmem
And it doesn't seem like either of these are 'official' distributions. 

From reading a bit more, my understanding is that xpmem is an optional feature to enable within the gni provider, and not a fabrics provider itself ?  Based on that, I have removed it from the libfabric fabrics list.

Reply:
Yes, it is referenced in ugni, but in this case it will come pre-installed with cray system. But as for now, I don't think it is a stand alone transport.

UCX has stanalone transport for XPMEM and I'm involved in https://github.com/hjelmn/xpmem as well.
XPMEM has both user level and kernel module components. So, I'm not sure if kernel modules are under spack scope.